### PR TITLE
Port ContentAlignmentEditor; add test for it

### LINKED ISSUE #2018: Port `System.Drawing.Design.ContentAlignmentEditor`
**Problem description:**

`ContentAlignmentEditor` from System.Drawing.Design is missing.

Refer to https://github.com/dotnet/winforms/issues/1115#issuecomment-501115599

### PR DESCRIPTION
**Pull request by zsd4yr**
closes #2018

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2117)

### CODE REVIEW

File: src/System.Windows.Forms.Design.Editors/src/System/Drawing/Design/ContentAlignmentEditor.ContentUI.cs [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+using System.ComponentModel.Design;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Windows.Forms.Design;
+
+namespace System.Drawing.Design
+{
+    public partial class ContentAlignmentEditor
+    {
+        /// <devdoc>
```
Comment (RussKie) [inline src/System.Windows.Forms.Design.Editors/src/System/Drawing/Design/ContentAlignmentEditor.ContentUI.cs]: All `devdocs` mustbe replaced with `summary`

File: src/System.Windows.Forms.Design.Editors/src/Misc/DpiHelper.cs [hunk 2]
```diff
@@ -142,6 +151,20 @@ public static int LogicalToDeviceUnits(int value, int devicePixels = 0)
             return (int)Math.Round(scalingFactor * (double)value);
         }
 
+        /// <summary>
+        /// Transforms a horizontal or vertical integer coordinate from logical to device units
+        /// by scaling it up  for current DPI and rounding to nearest integer value
+        /// </summary>
+        public static double LogicalToDeviceUnits(double value, int devicePixels = 0)
```
Comment (RussKie) [inline src/System.Windows.Forms.Design.Editors/src/Misc/DpiHelper.cs]: This is duplicate of https://github.com/dotnet/winforms/blob/release/3.1/src/Common/src/DpiHelper.cs#L212-L226

Comment (zsd4yr) [reply src/System.Windows.Forms.Design.Editors/src/Misc/DpiHelper.cs]: No it is not. The linked `LogicalToDeviceUnits` takes and returns an int.

The one required here takes and returns a double

Comment (zsd4yr) [reply src/System.Windows.Forms.Design.Editors/src/Misc/DpiHelper.cs]: I think the *right* think to do here is to remove all of the code from this class that uses `_pixelFactor` to do its own stuff and to let that be all handled by DPIHelper and other DPI-related scripts

**Comment by RussKie** [comment]
We can port it as is, and perform remediation separately
________________________________

**Pull request merged**